composer.json

```diff
@@ -13,6 +13,12 @@
       "Stev\\BTIPay\\": "src"
     }
   },
+  "autoload-dev": {
+    "psr-4": {
+      "Stev\\BTIPay\\Fixtures\\": "tests/fixtures",
```

Comment:
@gabrielsolomon , aceste fixtures sunt doar pentru teste. Nu ar fi mai ok sa stea tot sub namespaceul Stev\\BTIPay\\Tests\\Fixtures ?

Pentru ca in viitor poate vor fi si fixtures pentru aplicatie, poate un nomenclator cu ceva informatii sau cine stie...si atunci sa nu fie coliziune de namespace.

In mod normal la PSR-4 din ce-mi amintesc namespace-urile reflecta intocmai si structura directoarelor, nu ?